docs/reference/schema.md

```diff
@@ -44,7 +44,7 @@ The diagram below shows the core relationships between schema components, and th
       class Exposure{
         -Asset category
         -Taxonomy
-        -Cost type
+        -Metric type
```

Comment:
Mermaid diagrams will be replaced as part of https://github.com/GFDRR/rdl-standard/issues/178 so there's no need to update them